src/explore-education-statistics-admin/src/modules/find-statistics/components/KeyStatisticsSecondary.tsx

```diff
@@ -159,12 +159,12 @@ export const AddSecondaryStats = ({
                 ...release,
                 keyStatisticsSecondarySection,
               });
-              setIsPicking(false);
+              setIsFormOpen(false);
```

Comment:
Need to reset values?

Reply:
Not required. there isn't actually a form, it's just one input that will reset when it renders